src/styles/_card.scss

```
@@ -0,0 +1,75 @@
+.card {
+  width: 202px;
+  padding: 32px 16px 16px;
+  background-color: $fontColor__white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $fontColor__darkblue;
+  }
+
+  &__id-good {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $fontColor__lightgray;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $fontColor__lightgray;
+  }
+
+  &__price-text {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $fontColor__darkblue;
+  }
+
+  &__button {
+    text-align: center;
+    display: block;
+    margin-top: 16px;
+    box-sizing: border-box;
+    font-size: 14px;
+    line-height: 40px;
+    font-weight: 700;
+    background-color: $fontColor__blue;
+    color: $fontColor__white;
+    text-decoration: none;
+    border-radius: 5px;
+    border: 1px solid $fontColor__blue;
+    transition: 0.3s color linear, 0.3s background-color linear;
```

make a variable for transitionDuration
```suggestion
    transition: $transitionDuration color linear, $transitionDuration background-color linear;
```